src/github/folderRepositoryManager.ts

```diff
@@ -1540,6 +1540,9 @@ export const titleAndBodyFrom = (message: string): { title: string, body: string
 
 		body: idxLineBreak === -1
 			? ''
-			: message.slice(idxLineBreak + 1),
+			: message
+				.slice(idxLineBreak + 1)
+				.trim()
+				.replace(/\n+/g, ' '),

```

Comment:
This has the unfortunate side effect of causing my multiline commit message to have the newlines replaced with a space even when I have `"githubPullRequests.pullRequestDescription": "commit"` set. I think this is worse than the quick input box for `"custom"` having the newlines removed.

Reply:
Ah, yes. That is worse. Fixing that is possible, but requires refactoring the code. What is your current release timeframe that would include my original PR? If soon (please? 😉), I would prefer to hold off on fixing this PR to avoid any destabilization, especially given that this multi-line squashing behavior is pre-existing (again, though: I'm happy to address it).

Reply:
Let's wait until next week then, since we will likely release this week.

Reply:
I've resolved this. Please see the updated PR.